css/players.css

```diff
@@ -29,13 +30,16 @@ div#players div.eggs > *{
 display:inline-block;
 }
 div#players div.eggs:after {
-content: " ";
+c≠≠ontent: " ";
 display:block;
```

Comment:
Yep. Working remotely in nano, not really sure how that got in there.

-
larry williamson

music | http://lawrencealan.com/
work | http://tastestudio.com/
blog | http://r0b0tz.com

On Sun, Jul 31, 2016 at 9:34 PM, Max Leiter <notifications@github.com>
wrote:

> In css/players.css
> <https://github.com/OpenPoGo/OpenPoGoWeb/pull/150#discussion_r72916056>:
>
> > @@ -29,13 +30,16 @@ div#players div.eggs > *{
> >  display:inline-block;
> >  }
> >  div#players div.eggs:after {
> > -content: " ";
> > +c≠≠ontent: " ";
> >  display:block;
>
> I think the ≠ may be a mistake?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenPoGo/OpenPoGoWeb/pull/150/files/328a322cb738e2b751d9192ce770121912ebee30..631dc94f532135ada39004eb07bb544156220b70#r72916056>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJi_oI5KIOCVkpqwvDR5S4OFq4-K23qks5qbU0JgaJpZM4JZF6X>
> .
>